modules/virt-creating-a-vm-from-a-boot-source.adoc

```diff
@@ -1,28 +1,18 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
 
-[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
-= Creating a virtual machine from a default OS image
+[id="virt-creating-a-vm-from-a-boot-source_{context}"]
+= Creating a new virtual machine from an added boot source template
 
-After you have created a persistent volume claim (PVC) for default operating system (OS) images, you create a new virtual machine or virtual machine template from a default OS image that you uploaded to the PVC.
-
-[NOTE]
-====
-You can create virtual machines and virtual machine templates from default OS images with non-administrator user privileges. You need administrator privileges to upload and save default OS images to PVCs.
-====
-
-.Prerequisites
-
-* When you created the PVC, you selected the *Attach this data to a Virtual Machine operating system* check box.
+After you add a boot source to a template, you create a new virtual machine from the template.
 
 .Procedure
 
-. In the {product-title} web console, click *Workloads > Virtualization* in the sidebar menu. The *Virtualization* page displays. Use web console Help or existing documentation to create virtual machine templates.
-
-. Create a virtual machine or virtual machine template from an uploaded default OS image.
+. In the {product-title} web console, click *Workloads > Virtualization* in the sidebar menu. The *Virtualization* page displays.
+. Create a virtual machine or virtual machine template from a template with an attached boot source.
 .. Select *Create Virtual Machine > New with Wizard*. The *Create Virtual Machine* wizard displays.
-.. On the *General* wizard page, select an OS from the *Operating System* list that displays the *(Source available)* label next to the OS and version name. The *(Source available)* label indicates default OS images are available for this OS.
+.. On the *General* wizard page, select an OS from the *Operating System* list that displays the *(Source available)* label next to the OS and version name. The *(Source available)* label indicates if a boot source is available for this OS.
 .. Ensure that the *Clone available operating system source* check box is selected.
 .. Click the *Review and Confirm* button.
 .. On the *Review and confirm your settings* wizard page, review information about your virtual machine and change them, if required.
```

Comment:
Suggestion: Review your virtual machine settings and edit them, if required.

Reply:
Done.